metaci/conftest.py

```diff
@@ -18,7 +18,13 @@ class RepositoryFactory(factory.django.DjangoModelFactory):
     class Meta:
         model = Repository
 
+    name = factory.LazyAttribute(
+        lambda a: "Repo_"
+        + factory.Faker("paragraph").generate({}).replace(" ", "_")[0:20].lower()
```

Comment:
How about the `word` faker so we don't cut off mid-word?

Reply:
I'll take a look.